html/templates/header.html

```diff
@@ -1,20 +1,20 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
 <html>
 <head>
-  <title>Baystation 12 Changelog</title>
+  <title>TeguStation Changelog</title>
   <link rel="stylesheet" type="text/css" href="changelog.css">
   <base target="_blank" />
   <script type='text/javascript'>
-  
+

```

Comment:
Is this intentional?

Reply:
Visual Studio Code automatically linted orphaned indentations/spaces when editing the file. When copy-pasting the header into `changelog.html`, however, it works as normal. The screenshot you can see in the OP is from me doing that.